extensions/order-status/README.md

```diff
@@ -8,16 +8,18 @@ The `GET /orders/{orderId}` endpoint must add a link to this endpoint using the
 
 Endpoint: `GET /orders/{orderId}/statuses`
 
+TODO: Pagination
```

Comment:
Could it be sufficient to link to the [top level README pagination section](https://github.com/stapi-spec/stapi-spec?tab=readme-ov-file#pagination) along with a note that statuses should be paginated in accordance to those guidelines?